__tests__/app.test.js

```diff
@@ -58,3 +58,30 @@ afterAll(() => {
         });
       });
 
+      describe('GET /api/articles/:article_id', () => {
+        test('responds with an article object', () => {
+          return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then((response) => {
+              const article = response.body.article;
+              expect(article).toHaveProperty('author');
+              expect(article).toHaveProperty('title');
+              expect(article).toHaveProperty('article_id');
+              expect(article).toHaveProperty('body');
+              expect(article).toHaveProperty('topic');
+              expect(article).toHaveProperty('created_at');
+              expect(article).toHaveProperty('votes');
+              expect(article).toHaveProperty('article_img_url');
+            });
```

Comment:
As we're checking only 1 article, I'd like to see this test refactored to check the actual contents of this article, i.e. what is the value of author? what is the value of title? etc